src/homepageExperience/components/steps/cli/InitializeClient.tsx

```diff
@@ -42,22 +54,23 @@ export const InitializeClient: FC<OwnProps> = ({
   wizardEventName,
   setTokenValue,
   tokenValue,
+  onSelectBucket,
 }) => {
   const org = useSelector(getOrg)
   const me = useSelector(getMe)
   const allPermissionTypes = useSelector(getAllTokensResources)
   const dispatch = useDispatch()
   const url =
     me.quartzMe?.clusterHost || 'https://us-west-2-1.aws.cloud2.influxdata.com/'
-  const orgID = org.id
+  const orgName = org.name
```

Comment:
This `orgID` and `orgName` are coming from IDPE land. Could make sense to have these fetched using a `quartz` selector, since this component lives in the cloud, and that'll be a better way of tracking the currently active org long-term.